scripts/ldev2pcs

```diff
@@ -137,6 +137,10 @@ def configure():
         cmd += ' pool='+zpool
         cmd += ' op start timeout=805'
         run(cmd)
+
+        # We want suspended pools to immediately fail over to the other node.
+        run('pcs resource meta '+zpool_resource+' migration-threshold=1')
```

Comment:
I believe that this can be added to the 'pcs resource create' line instead of being a separate command, but this will work fine too.